### DISCUSSION TITLE
How to disable (certain) hotkeys

### DISCUSSION THREAD

**Discussion by BetweenBracketsCv** [react-page/react-page#1305] [Q&A]
Hi,

We are not using slate, but our own wysiwyg editor. However when users use `ctrl-z` the editor will undo the last action. Is there a way to disable (certain) hotkeys baked into the editor?

**Comment by macrozone** [comment]
> Hi,
> 
> We are not using slate, but our own wysiwyg editor. However when users use `ctrl-z` the editor will undo the last action. Is there a way to disable (certain) hotkeys baked into the editor?

ctrl+z is disabled in slate, because we handle ctrl-z to undo / redo ALL cell manipulations, not only from rich-text editor.

So you should:

- disable ctrl+z in your customer wysiwyg Cell plugin
- make sure the Cell plugin properly updates it state based on the cell's data
